app/src/main/java/services/authhandler/AuthHandler.java

```diff
@@ -0,0 +1,66 @@
+package services.authhandler;
+
+import android.util.Log;
+
+import com.amplifyframework.auth.AuthUserAttributeKey;
+import com.amplifyframework.auth.options.AuthSignUpOptions;
+import com.amplifyframework.core.Amplify;
+
+public class AuthHandler{
+
+    public static void signUp(String email, String username, String password, AuthEvents eventStatus){
+        AuthSignUpOptions options = AuthSignUpOptions.builder()
```

Comment:
Fix indentations

Reply:
This the default, as well as auto code reformat, indentation cause they are parameters to a function.

Reply:
Tweaked it anyways